pydantic/main.py

```diff
@@ -1322,11 +1316,13 @@ def create_model(
 
 
 def _collect_bases_data(bases: tuple[type[Any], ...]) -> tuple[set[str], set[str], dict[str, ModelPrivateAttr]]:
+    if bases == (_BaseModelBase,):
+        bases = ()
```

Comment:
```suggestion
```
It seems to me this is unnecessary since the only place where this is called already checks for this conditional and bails out. (And all tests pass for me locally without these two lines.)

Reply:
Yes. I was just a bit paranoid. And there was an old check there. So, I'm not oppose to removing this but would like to have a some kind of a check here. Maybe an `assert` like `assert(bases != (_BaseModelBase,))`?

Reply:
BTW, this could be safely removed if `BaseModel` has no bases.